src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Language.txt

```diff
@@ -0,0 +1,2 @@
+>> Language()
+"en-US"
```

Comment:
Can we get a different result if someone in France run this test?

Reply:
No, we have default cultrure set for test is en-Us